devel/design/networking-api.md

```diff
@@ -514,7 +514,7 @@ interfaces:
   state: up
   ipv4:
     address:
-    - ip: 10.10.10.10
+    - ip: 192.0.2.0
```

Comment:
Hello! I think this is also needed in other IP examples on the documentation. Could you do it? Thank you! :-)